src/features/Authentication/authenticationApi.ts

```diff
@@ -1,92 +1,203 @@
+import * as D from 'io-ts/Decoder';
 import { createApi } from '@reduxjs/toolkit/query/react';
+import { pipe } from 'fp-ts/lib/function';
+import { t } from 'i18next';
+import toast from 'react-hot-toast';
+
 import { parseAndTransformTo, refreshingBaseQuery } from '@/utils/http';
-import * as D from 'io-ts/Decoder';
 
 export const role = D.literal('mentee', 'mentor', 'admin');
 
 const userCodec = D.struct({
   account_id: D.string,
   active: D.boolean,
-  created: D.string,
   display_name: D.string,
   id: D.string,
   role: role,
-  updated: D.string,
 });
 
 const accountCodec = D.struct({
   active: D.boolean,
-  created: D.string,
   email: D.string,
   id: D.string,
   login_name: D.string,
   role: role,
-  updated: D.string,
 });
 
-const myuserResponse = D.struct({
+const mentorCodec = D.struct({
+  account_id: D.string,
+  active: D.boolean,
+  birth_year: D.number,
+  communication_channels: D.array(D.string),
+  display_name: D.string,
+  gender: D.string,
+  id: D.string,
+  is_vacationing: D.boolean,
+  languages: D.array(D.string),
+  region: D.string,
+  skills: D.array(D.string),
+  status_message: D.string,
+  story: D.string,
+  user_id: D.string,
+});
```

Comment:
We alreaady have mentorCodec defined in mentorPageApi. Instead we could use the same one?

Reply:
If there is other codecs that are duplicates in this PR, I maybe dont see them. But I would prefer to use existing ones if there is. 
 => One source of truth instead of many different definitiions

Reply:
I thought about combining these, but they've got some different fields so I didn't yet.

Reply:
Thought it might introduce some trouble.

Reply:
Well actually it was quick and easy so it's done now! 😃

Reply:
Well actually it wasn't that bad so it's done now! 😃